test/csuite/import/smoke.sh

```diff
@@ -63,7 +63,7 @@ import()
 	cp $rundir/wt $foreign/yyy
 
 	# Import the table.
-	$wt -C "$EXT" -h $foreign import file:yyy
+	$wt -C "$EXT" -h $foreign create -c 'import=(enabled,repair=true)' file:yyy
```

Comment:
Do we want to maintain this file? There doesn't seem to be any benefit in running an additional csuite test just for the repair case. I'm leaning towards removing it.

Reply:
I think it's useful because it's testing that we can import via the `wt` command line tool (as opposed to the session API).
I can be convinced either way though.

Reply:
Do we usually test other API functions and configurations with the `wt` tool?

Reply:
There are some tests that exercise functionality via `wt` but the coverage is patchy. I think most of them are under `test_utilXX`. Not sure why this one is a csuite test.

Actually, one of my pain points with `wt` is that the testing is insufficient and functionality often gets broken by library changes (WT-6354). This one is probably less prone to breakage though since the mapping to the `session->create` call is trivial.

Reply:
As discussed, I'll move this testing over to `test_util03`.

Reply:
Removing the csuite test: 70c4c10
Adding the new test in `util03`: d9aad7f